lightly/loss/sym_neg_cos_sim_loss.py

```diff
@@ -0,0 +1,62 @@
+""" Symmetrized Negative Cosine Similarity Loss Functions """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+import torch
+
+class SymNegCosineSimilarityLoss(torch.nn.Module):
+    """Implementation of the Symmetrized Loss.
+    
+    Examples:
+
+        >>> # initialize loss function
+        >>> loss_fn = SymmetrizedLoss
+        >>>
+        >>> # generate two random transforms of images
+        >>> t0 = transforms(images)
+        >>> t1 = transforms(images)
+        >>>
+        >>> # feed through SimSiam model
+        >>> output = model(t0, t1)  
+        >>>
+        >>> # calculate loss
+        >>> loss = loss_fn(output)
+
```

Comment:
The example is not up-to-date, it should be:
```python
>>> out0, out1 = model(t0, t1)
>>> loss = loss_fn(out0, out1)
```

and you're missing the brackets for the constructor:
```python
>>> loss_fn = SymmetrizedLoss()
```

Reply:
Good catch!